firestore.rules

```diff
@@ -46,25 +46,25 @@ service cloud.firestore {
     }
 
     // uid of submitted document must match user's platform_user_id
-    function requestMatchUserId() {
+    function userIsRequestUser() {
```

Comment:
Would it be preferable to have all the functions, that return booleans, consistently use `isXabcYdg()` / `hasZcba()` naming? E.g., `isUserRequestUser()`, in this case?

Reply:
I've gone back and forth on that convention over the years. In some cases I think it just reads better the other way and I'm not convinced the consistency adds measurable value so I'm content to let it be developer discretion (mine in this case).